modules/icgc-argo-workflows/prep_sample/resources/usr/bin/main.py

```diff
@@ -250,7 +248,12 @@ def main():
     donor_id = song_analysis['samples'][0]['donor']['donorId']
     sample_id = song_analysis['samples'][0]['sampleId']
     specimen_id = song_analysis['samples'][0]['specimen']['specimenId']
-    sex = 'XX' if song_analysis['samples'][0]['donor']['gender'] == 'Female' else 'XY' 
+    if song_analysis['samples'][0]['donor']['gender'] == 'Female':
+       sex = 'XX'  
+    elif song_analysis['samples'][0]['donor']['gender'] == 'Male': 
+       sex = 'XY'
+    else:
+       sex = 'Other'
```

Comment:
Should value be [NA](https://nf-co.re/sarek/3.1.2/usage#overview-samplesheet-columns) instead?

Reply:
Yes. should set it as `NA` to be compatible with other nf-core modules. 👍